reference/latest/src/main/resources/fabric.mod.json

```diff
@@ -19,16 +19,19 @@
 		"com.example.docs.block.FabricDocsReferenceBlocks",
 		"com.example.docs.block.entity.FabricDocsReferenceBlockEntities",
 		"com.example.docs.component.FabricDocsReferenceComponents",
-		"com.example.docs.networking.FabricDocsReferenceNetworking"
+		"com.example.docs.networking.FabricDocsReferenceNetworking",
+		  "com.example.docs.networking.basic.FabricDocsReferenceNetworkingBasic"
 	  ],
 	  "client": [
 		"com.example.docs.FabricDocsReferenceClient",
 		"com.example.docs.client.command.FabricDocsReferenceClientCommands",
-		"com.example.docs.FabricDocsDynamicSound"
+		"com.example.docs.FabricDocsDynamicSound",
+		  "com.example.docs.network.basic.FabricDocsReferenceNetworkingBasicClient"
 	  ],
 	  "fabric-datagen": [
 		"com.example.docs.FabricDocsReferenceDataGenerator",
-		"com.example.docs.damage.FabricDocsReferenceDamageTypesDataGenerator"
+		"com.example.docs.damage.FabricDocsReferenceDamageTypesDataGenerator",
+		"com.example.docs.networking.basic.FabricDocsReferenceNetworkingBasicDataGenerator"
```

Comment:
You cant have multiple data generators, it doesnt work. Please use the main datagenerator (FabricDocsReferenceDataGenerator)

Reply:
Move it into the main sourceset if required.

Reply:
If you do add a provider to the main data generator, add it to the end of the list, some pages are referencing the file by line number to show the `pack.addProvider` call. 

Example:

![chrome_bVhPXuS4Yq](https://github.com/user-attachments/assets/d620bdff-490a-4636-bcf4-e0af507c9ef8)